hub/core/compression/numpy.py

```diff
@@ -0,0 +1,47 @@
+from io import BytesIO
+
+import numpy as np
+from hub.core.compression import BaseNumCodec
+
+
+class NUMPY(BaseNumCodec):
```

Comment:
Does this even compress the data? I thought pickle just converted stuff into bytes.

Reply:
It does, but the binary representation it uses is better than serializing plain python objects. I think that counts as compression of a sort